src/ext.js

```diff
@@ -155,19 +155,18 @@ export default function ext({ translator, shouldHide, senseNavigation }) {
                 },
                 // adds automation to actions and adds a dropdown property panel
                 // item to select the automation for the button to trigger
-                automation: {
+                automationId: {
                   type: 'string',
                   component: 'expression-with-dropdown',
                   translation: 'Object.ActionButton.Automation',
-                  ref: 'automation',
+                  ref: 'automationId',
```

Comment:
changing the ref here will break all buttons already using this property. If the intention is to replace the old action then you need to make sure old objects that has `automation` saved are mapped correctly.

Reply:
Hi @Caele 

I do handle the migration of this property to the new property here:
https://github.com/qlik-oss/sn-action-button/blob/31a1055794e9515bf87be1d2c22d94ad85f1b4a3/src/index.js#L70-L124

I also ensure if the property has not been migrated that it will still work here:
https://github.com/qlik-oss/sn-action-button/blob/31a1055794e9515bf87be1d2c22d94ad85f1b4a3/src/utils/actions.js#L304-L307

Reply:
Still you'd want to hide the new functionality behind a feature flag, so want to keep the old code as well as the new?

Reply:
Feature flag has now been added